<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Wen Yao</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Publications.html">Publications</a>
</li>
<li>
  <a href="Tools.html">Tools</a>
</li>
<li>
  <a href="Links.html">Links</a>
</li>
<li>
  <a href="Contact.html">Contact</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<div id="section" class="section level4">
<h4>2022</h4>
<ul>
<li>Fangfang Huang<sup>+</sup>, Yingru Jiang<sup>+</sup>, Tiantian
Chen<sup>+</sup>, Haoran Li, Mengjia Fu, Yazhou Wang, Yufang Xu, Yang
Li, Zhengfu Zhou, Lihua Jia, Yidan Ouyang, <strong>Wen
Yao</strong><sup>✉</sup>.
<a href="https://doi.org/10.1186/s12284-022-00569-1" target="_blank">New
data and new features of the funRiceGenes (functionally characterized
rice genes) database: 2021 update</a>. <strong>Rice</strong>, 2022</li>
<li>Zhengfu Zhou, Shenghui Geng, Huiyue Guan, Congcong Liu, Maomao Qin,
Wenxu Li, Xia Shi, Ziju Dai, <strong>Wen Yao</strong>, Zhensheng
Lei<sup>✉</sup>, Zhengqing Wu<sup>✉</sup>, Jinna Hou<sup>✉</sup>.
<a href="https://doi.org/10.3389/fpls.2022.826909" target="_blank">Dissection
of the genetic architecture for quantities of gliadins fractions in
wheat (Triticum aestivum L.)</a>. <strong>Frontiers in Plant
Science</strong>, 2022</li>
</ul>
</div>
<div id="section-1" class="section level4">
<h4>2021</h4>
<ul>
<li>Lihua Jia<sup>+</sup>, Yang Li<sup>+</sup>, Fangfang Huang, Yingru
Jiang, Haoran Li, Zhizhan Wang, Tiantian Chen, Jiaming Li, Zhang
Zhang<sup>✉</sup>, <strong>Wen Yao</strong><sup>✉</sup>.
<a href="https://doi.org/10.1093/nar/gkab912" target="_blank">LIRBase: a
comprehensive database of long inverted repeats in eukaryotic
genomes</a>. <strong>Nucleic Acids Research</strong>, 2021</li>
<li>Lihua Jia<sup>+</sup>, <strong>Wen
Yao</strong><sup>+</sup><sup>,</sup><sup>✉</sup>, Yingru Jiang, Yang Li,
Zhizhan Wang, Haoran Li, Fangfang Huang, Jiaming Li, Tiantian Chen,
Huiyong Zhang<sup>✉</sup>.
<a href="https://doi.org/10.1093/bib/bbab415" target="_blank">Development
of interactive biological web applications with R/Shiny</a>.
<strong>Briefs in Bioinformatics</strong>, 2021</li>
<li>Huaisheng Zhang<sup>+</sup>, Huili Yang<sup>+</sup>, Desheng Hu,
Bing Li, Yanan Lin, <strong>Wen Yao</strong>, Zhanyong Guo, Haochuan Li,
Dong Ding, Zhanhui Zhang, Yanmin Hu, Yadong Xue<sup>✉</sup>, Jihua
Tang<sup>✉</sup>.
<a href="https://doi.org/10.1016/j.cj.2021.06.012" target="_blank">Single-cell
RNA sequencing of meiocytes and microspores reveals the involvement of
the Rf4 gene in redox homeostasis of CMS-C maize</a>. <strong>The Crop
Journal</strong>, 2021</li>
<li>Zhengfu Zhou<sup>+</sup>, Congcong Liu<sup>+</sup>, Maomao Qin,
Wenxu Li, Jinna Hou, Xia Shi, Ziju Dai, <strong>Wen Yao</strong>,
Baoming Tian, Zhensheng Lei<sup>✉</sup>, Yang Li<sup>✉</sup>, Zhengqing
Wu<sup>✉</sup>.
<a href="https://doi.org/10.1016/j.jare.2021.06.021" target="_blank">Promoter
DNA hypermethylation of TaGli-γ-2.1 positively regulates gluten strength
in bread wheat</a>. <strong>Journal of Advanced Research</strong>,
2021</li>
<li>Chen You<sup>+</sup>, YuPing Zhang<sup>+</sup>, ShaoYu
Yang<sup>+</sup>, Xu Wang, <strong>Wen Yao</strong>, WeiHuan Jin, Wei
Wang, XiuLi Hu, Hao Yang<sup>✉</sup>.
<a href="https://doi.org/10.3389/fpls.2021.641517" target="_blank">Proteomic
Analysis of Generative and Vegetative Nuclei Reveals Molecular
Characteristics of Pollen Cell Differentiation in Lily</a>.
<strong>Frontiers in Plant Science</strong>, 2021</li>
<li>Jeannie Chan<sup>✉</sup>, <strong>Wen Yao</strong>, Timothy D
Howard, Gregory A Hawkins, Michael Olivier, Matthew J Jorgensen, Ian H
Cheeseman, Shelley A Cole, Laura A Cox.
<a href="https://doi.org/10.1111/jmp.12524" target="_blank">Efficiency
of whole-exome sequencing in old world and new world primates using
human capture reagents</a>. <strong>Journal of Medical
Primatology</strong>, 2021</li>
<li>Xiaona Cui<sup>+</sup>, Kaili Gao<sup>+</sup>, Linjuan
Wang<sup>+</sup>, Mengyang Lv, Ziwen Li, Donghua Zheng, Wenwu Wu,
<strong>Wen Yao</strong>, Liying Ding, Xiao Li, Jian-Kang Zhu, Hairong
Zhang<sup>✉</sup>.
<a href="https://www.frontiersin.org/articles/10.3389/fpls.2021.630311/full" target="_blank">General
Control Non-derepressible 1 (AtGCN1) Is Important for Flowering Time,
Plant Growth, Seed Development, and the Transcription/Translation of
Specific Genes in <em>Arabidopsis</em></a>. <strong>Frontiers in Plant
Science</strong>, 2021</li>
<li>Zhengfu Zhou<sup>+</sup>, Ziwei Zhang<sup>+</sup>, Lihua
Jia<sup>+</sup>, Hongxia Qiu, Huiyue Guan, Congcong Liu, Maomao Qin,
Yahuan Wang, Wenxu Li, <strong>Wen Yao</strong>, Zhengqing
Wu<sup>✉</sup>, Baoming Tian<sup>✉</sup>, Zhensheng Lei<sup>✉</sup>.
<a href="https://doi.org/10.3389/fpls.2020.611605" target="_blank">Genetic
Basis of Gluten Aggregation Properties in Wheat (<em>Triticum
aestivum</em> L.) Dissected by QTL Mapping of GlutoPeak Parameters</a>.
<strong>Frontiers in Plant Science</strong>, 2021</li>
</ul>
</div>
<div id="section-2" class="section level4">
<h4>2020</h4>
<ul>
<li><strong>Wen Yao</strong><sup>✉</sup>, Yang Li, Weibo Xie, Lei Wang.
<a href="https://doi.org/10.1016/j.csbj.2020.10.012" target="_blank">Features
of sRNA biogenesis in rice revealed by genetic dissection of sRNA
expression level</a>. <strong>Computational and Structural Biotechnology
Journal</strong>, 2020</li>
<li>Congcong Liu<sup>+</sup>, Zhengfu Zhou<sup>+</sup>, Zhengqing Wu,
Maomao Qin, Xia Shi, Yahuan Wang, Wenxu Li, <strong>Wen
Yao</strong><sup>✉</sup>, Zhensheng Lei<sup>✉</sup>, Renyong
Zhao<sup>✉</sup>.
<a href="https://www.sciencedirect.com/science/article/pii/S0888754320307631" target="_blank">In-depth
genetic analysis reveals conditioning of polyphenol oxidase activity in
wheat grains by <em>cis</em> regulation of <em>TaPPO2A-1</em> expression
level</a>. <strong>Genomics</strong>, 2020 (<sup>+</sup> co-first
author)<br />
</li>
<li>Xin Wang<sup>+</sup>, Xin Yan<sup>+</sup>, Xiaoxiao
Tian<sup>+</sup>, Zongfei Zhang, Weiwei Wu, Junjun Shang, Jiexiu Ouyang,
<strong>Wen Yao</strong><sup>✉</sup>, Shaobo Li<sup>✉</sup>.
<a href="https://pubs.acs.org/doi/abs/10.1021/acs.jafc.0c01803" target="_blank">Glycine-and
proline-rich protein OsGPRP3 regulates grain size and quality in
rice</a>. <strong>Journal of Agricultural and Food Chemistry</strong>,
2020 (<sup>+</sup> co-first author)<br />
</li>
<li>Jia L<sup>+</sup>, Liu N<sup>+</sup>, Huang F, Zhou Z, He X, Li H,
Wang Z, <strong>Yao W</strong><sup>✉</sup>.
<a href="https://peerj.com/articles/8867/" target="_blank">intansv: an R
package for integrative analysis of structural variations</a>.
<strong>PeerJ</strong>, 2020 (<sup>+</sup> co-first author)<br />
</li>
<li>Zijian Zhou<sup>+</sup>, Yan Cao<sup>+</sup>, Tao Li, Xinghao Wang,
Jiafa Chen, Hang He, <strong>Wen Yao</strong>, Jianyu Wu<sup>✉</sup>,
Huiyong Zhang<sup>✉</sup>.
<a href="https://www.sciencedirect.com/science/article/pii/S1672022920300589" target="_blank">MicroRNAs
Are Involved in Maize Immunity Against Fusarium verticillioides Ear
Rot</a>. <strong>Genomics, Proteomics &amp; Bioinformatics</strong>,
2020 (<sup>+</sup> co-first author)</li>
<li>Sheng Zhao<sup>+</sup>, Cuicui Zhang<sup>+</sup>, Jianqiang Mu, Hui
Zhang, <strong>Wen Yao</strong>, Xinhua Ding, Junqiang Ding<sup>✉</sup>,
Yuxiao Chang<sup>✉</sup>.
<a href="https://plantmethods.biomedcentral.com/articles/10.1186/s13007-020-00615-3" target="_blank">All-in-one
sequencing: an improved library preparation method for cost-effective
and high-throughput next-generation sequencing</a>. <strong>Plant
Methods</strong>, 2020 (<sup>+</sup> co-first author)</li>
</ul>
</div>
<div id="section-3" class="section level4">
<h4>2019</h4>
<ul>
<li>Yu Y<sup>+</sup>, <strong>Yao
W</strong><sup>+</sup><sup>,</sup><sup>✉</sup>, Wang Y, Huang F.
<a href="https://doi.org/10.1016/j.gpb.2019.07.003" target="_blank">shinyChromosome:
An R/Shiny Application for Interactive Creation of Non-circular Plots of
Whole Genomes</a>. <strong>Genomics, Proteomics &amp;
Bioinformatics</strong>, 2019 (<sup>+</sup> co-first author)</li>
<li>Zhou W, Wang L, Zheng W<sup>✉</sup>, <strong>Yao
W</strong><sup>✉</sup>.
<a href="https://doi.org/10.1016/j.csbj.2019.10.003" target="_blank">MaizeSNPDB:
a comprehensive database for efficient retrieve and analysis of SNPs
among 1210 maize lines</a>. <strong>Computational and Structural
Biotechnology Journal</strong>, 2019<br />
</li>
<li><strong>Yao W</strong><sup>+</sup>, Li G<sup>+</sup>, Cui Y, Yu Y,
Zhang Q<sup>✉</sup>, Xu S<sup>✉</sup>.
<a href="https://www.sciencedirect.com/science/article/pii/S167385271930116X" target="_blank">Mapping
quantitative trait loci using binned genotypes</a>. <strong>Journal of
Genetics and Genomics</strong>, 2019 (<sup>+</sup> co-first
author)<br />
</li>
<li><strong>Yao W</strong><sup>✉</sup>, Huang F, Zhang X, Tang
J<sup>✉</sup>.
<a href="https://doi.org/10.1093/bioinformatics/btz186" target="_blank">ECOGEMS:
efficient compression and retrieve of SNP data of 2058 rice accessions
with integer sparse matrices</a>. <strong>Bioinformatics</strong>,
2019<br />
</li>
<li>Peter Brown, RELISH Consortium, Yaoqi Zhou.
<a href="https://doi.org/10.1093/database/baz085" target="_blank">Large
expert-curated database for benchmarking document similarity detection
in biomedical literature search</a>. <strong>Database</strong>,
2019</li>
<li>Luo X, Li H, Wu Z, <strong>Yao W</strong>, Zhao P, Cao D, Yu H, Li
K, Poudel K, Zhao D, Zhang F, Xia X, Chen L, Wang Q, Jing D, Cao S.
<a href="https://doi.org/10.1111/pbi.13260" target="_blank">The
pomegranate (Punica granatum L.) draft genome dissects genetic
divergence between soft- and hard-seeded cultivars</a>. <strong>Plant
Biotechnology Journal</strong>, 2019</li>
<li>Wang X<sup>+</sup>, Yu C<sup>+</sup>, Liu Y, Yang L, Li Y,
<strong>Yao W</strong>, Cai Y, Yan X, Li S, Cai Y, Li S, Peng
X<sup>✉</sup>.
<a href="https://www.mdpi.com/1422-0067/20/15/3796/htm" target="_blank">GmFAD3A,
A ω-3 Fatty Acid Desaturase Gene, Enhances Cold Tolerance and Seed
Germination Rate under Low Temperature in Rice</a>. <strong>Int. J. Mol.
Sci.</strong>, 2019 (<sup>+</sup> co-first author)</li>
</ul>
</div>
<div id="section-4" class="section level4">
<h4>2018</h4>
<ul>
<li>Su H<sup>+</sup>, Cao Y<sup>+</sup>, Ku
L<sup>+</sup><sup>,</sup><sup>✉</sup>, <strong>Yao W</strong>, Cao Y,
Ren Z, Dou D, Wang H, Ren Z, Liu H, Tian L, Zheng Y, Chen C, Chen
Y<sup>✉</sup>.
<a href="https://doi.org/10.1093/jxb/ery299" target="_blank">Dual
functions of ZmNF-YA3 in photoperiod-dependent flowering and abiotic
stress responses in maize</a>. <strong>J Exp Bot</strong>, 2018
(<sup>+</sup> co-first author)</li>
</ul>
</div>
<div id="section-5" class="section level4">
<h4>2017</h4>
<ul>
<li><strong>Yao W</strong><sup>✉</sup>, Li G, Yu Y, Ouyang
Y<sup>✉</sup>.
<a href="https://academic.oup.com/gigascience/article/7/1/gix119/4689117" target="_blank">funRiceGenes
dataset for comprehensive understanding and application of rice
functional genes</a>. <strong>GigaScience</strong>, 2017<br />
</li>
<li>Yu Y, Ouyang Y<sup>✉</sup>, <strong>Yao W</strong><sup>✉</sup>.
<a href="https://doi.org/10.1093/bioinformatics/btx763" target="_blank">shinyCircos:
an R/Shiny application for interactive creation of Circos plot</a>.
<strong>Bioinformatics</strong>, 2017</li>
</ul>
</div>
<div id="section-6" class="section level4">
<h4>2016</h4>
<ul>
<li>Dong H, Zhao H, Xie W, Han Z, Li G, <strong>Yao W</strong>, Bai X,
Hu Y, Guo Z, Lu K, Yang L, Xing Y<sup>✉</sup>.
<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006412" target="_blank">A
Novel Tiller Angle Gene, <em>TAC3</em>, together with <em>TAC1</em> and
<em>D2</em> Largely Determine the Natural Variation of Tiller Angle in
Rice Cultivars</a>. <strong>PLoS Genetics</strong>, 2016<br />
</li>
<li>Zhang J<sup>+</sup>, Chen L-L<sup>+</sup>, Xing F<sup>+</sup>,
Kudrna DA, <strong>Yao W</strong>, Copetti D, Mu T, Li W, Song J-M, Xie
W, Lee S, Talag J, Shao L, An Y, Zhang C-L, Ouyang Y, Sun S, Jiao W-B,
Lv F, Du B, Luo M, Maldonado C, Goicoechea J, Xiong L, Wu C, Xing Y,
Zhou D, Yu S, Zhao Y, Wang G, Yu Y, Luo Y, Zhou Z, Hurtado B, Danowitz
A, Wing R<sup>✉</sup>, Zhang Q<sup>✉</sup>.
<a href="http://www.pnas.org/content/113/35/E5163.full" target="_blank">Extensive
sequence divergence between the reference genomes of two elite indica
rice varieties Zhenshan 97 and Minghui 63</a>. <strong>Proc Natl Acad
Sci U S A</strong>, 2016 (<sup>+</sup> co-first author)<br />
</li>
<li>Magwa R, Zhao H, <strong>Yao W</strong>, Xie W, Yang L, Xing Y, Bai
X<sup>✉</sup>.
<a href="https://link.springer.com/article/10.1007/s12041-016-0679-1" target="_blank">Genome-wide
association analysis for awn length linked to the seed shattering gene
<em>qSH1</em> in rice</a>. <strong>J Genet</strong>, 2016<br />
</li>
<li>Li Q, Yan W, Chen H, Tan C, Han Z, <strong>Yao W</strong>, Li G,
Yuan M, Xing Y<sup>✉</sup>.
<a href="https://doi.org/10.1093/jxb/erv566" target="_blank">Duplication
of <em>OsHAP</em> family genes and their association with heading date
in rice</a>. <strong>J Exp Bot</strong>, 2016</li>
</ul>
</div>
<div id="section-7" class="section level4">
<h4>2015</h4>
<ul>
<li><strong>Yao W</strong>, Li G, Zhao H, Wang G, Lian X, Xie
W<sup>✉</sup>.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0757-3" target="_blank">Exploring
the rice dispensable genome using a metagenome-like assembly
strategy</a>. <strong>Genome Biology</strong>, 2015<br />
</li>
<li>Wang J<sup>+</sup>, <strong>Yao W</strong><sup>+</sup>, Zhu D, Xie
W, Zhang Q<sup>✉</sup>.
<a href="https://elifesciences.org/articles/03913" target="_blank">Genetic
basis of sRNA quantitative variation analyzed using an experimental
population derived from an elite rice hybrid</a>.
<strong>eLife</strong>, 2015 (<sup>+</sup> co-first author)<br />
</li>
<li>Zhao H, <strong>Yao W</strong>, Ouyang Y, Yang W, Wang G, Lian X,
Xing Y, Chen L, Xie W<sup>✉</sup>.
<a href="https://academic.oup.com/nar/article/43/D1/D1018/2439488/RiceVarMap-a-comprehensive-database-of-rice" target="_blank">RiceVarMap:
a comprehensive database of rice genomic variations</a>. <strong>Nucleic
Acids Res</strong>, 2015<br />
</li>
<li>Zhang L<sup>+</sup>, Mao D<sup>+</sup>, Xing F, Bai X, Zhao H,
<strong>Yao W</strong>, Li G, Xie W, Xing Y<sup>✉</sup>.
<a href="http://www.sciencedirect.com/science/article/pii/S0168945215001739" target="_blank">Loss
of function of <em>OsMADS3</em> via the insertion of a novel
retrotransposon leads to recessive male sterility in rice (<em>Oryza
sativa</em>)</a>. <strong>Plant Sci</strong>, 2015 (<sup>+</sup>
co-first author)<br />
</li>
<li>Zhang L, Li Q, Dong H, He Q, Liang L, Tan C, Han Z, <strong>Yao
W</strong>, Li G, Zhao H, Xie W, Xing Y<sup>✉</sup>.
<a href="https://www.nature.com/articles/srep07663" target="_blank">Three
CCT domain-containing genes were identified to regulate heading date by
candidate gene-based association mapping and transformation in rice</a>.
<strong>Sci Rep</strong>, 2015<br />
</li>
<li>Yang Z, Ma H, Hong H, <strong>Yao W</strong>, Xie W, Xiao J, Li X,
Wang S<sup>✉</sup>.
<a href="https://thericejournal.springeropen.com/articles/10.1186/s12284-014-0038-x" target="_blank">Transcriptome-based
analysis of mitogen-activated protein kinase cascades in the rice
response to Xanthomonas oryzae infection</a>. <strong>Rice</strong>,
2015<br />
</li>
<li>Xie W<sup>+</sup>, Wang G<sup>+</sup>, Yuan M, <strong>Yao
W</strong>, Lyu K, Zhao H, Yang M, Li P, Zhang X, Yuan J, Wang Q, Liu F,
Dong H, Zhang L, Li X, Meng X, Zhang W, Xiong L, He Y, Wang S, Yu S, Xu
C, Luo J, Li X, Xiao J, Lian X<sup>✉</sup>, Zhang Q<sup>✉</sup>.
<a href="http://www.pnas.org/content/112/39/E5411.full" target="_blank">Breeding
signatures of rice improvement revealed by a genomic variation map from
a large germplasm collection</a>. <strong>Proc Natl Acad Sci U S
A</strong>, 2015 (<sup>+</sup> co-first author)</li>
</ul>
</div>
<div id="section-8" class="section level4">
<h4>2014</h4>
<ul>
<li>Peng B<sup>+</sup>, Kong H<sup>+</sup>, Li Y, Wang L, Zhong M, Sun
L, Gao G, Zhang Q, Luo L, Wang G, Xie W, Chen J, <strong>Yao W</strong>,
Peng Y, Lei L, Lian X, Xiao J, Xu C, Li X, He Y<sup>✉</sup>.
<a href="https://www.nature.com/articles/ncomms5847" target="_blank"><em>OsAAP6</em>
functions as an important regulator of grain protein content and
nutritional quality in rice</a>. <strong>Nat Commun</strong>, 2014
(<sup>+</sup> co-first author)</li>
</ul>
</div>
<div id="section-9" class="section level4">
<h4>2013</h4>
<ul>
<li>Yan W<sup>+</sup>, Liu H<sup>+</sup>, Zhou X, Li Q, Zhang J, Lu L,
Liu T, Liu H, Zhang C, Zhang Z, Shen G, <strong>Yao W</strong>, Chen H,
Yu S, Xie W, Xing Y<sup>✉</sup>.
<a href="https://www.nature.com/articles/cr201343" target="_blank">Natural
variation in <em>Ghd7.1</em> plays an important role in grain yield and
adaptation in rice</a>. <strong>Cell Res</strong>, 2013 (<sup>+</sup>
co-first author)<br />
</li>
<li>Wang N, Long T, <strong>Yao W</strong>, Xiong L, Zhang Q, Wu
C<sup>✉</sup>.
<a href="http://www.sciencedirect.com/science/article/pii/S1674205214600253" target="_blank">Mutant
resources for the functional analysis of the rice genome</a>.
<strong>Mol Plant</strong>, 2013</li>
</ul>
</div>
<div id="section-10" class="section level4">
<h4>2012</h4>
<ul>
<li>Zhou G, Chen Y, <strong>Yao W</strong>, Zhang C, Xie W, Hua J, Xing
Y, Xiao J, Zhang Q<sup>✉</sup>.
<a href="http://www.pnas.org/content/109/39/15847.long" target="_blank">Genetic
composition of yield heterosis in an elite rice hybrid</a>. <strong>Proc
Natl Acad Sci U S A</strong>, 2012</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
